test/db/formats/dmp/dmp

```diff
@@ -98,3 +98,249 @@ EXPECT=<<EOF
 0x00001030  0000 0000 0000 0000 0000 0000 0000 0000  ................
 EOF
 RUN
+
+NAME=: Triage Dump Debug x64
+FILE=dmp://bins/dmp/triage_x64.dmp
+ARGS=-d
+CMDS=<<EOF
+iH
+e asm.sub.names=false
+pi 4
+dr
```

Comment:
Can you also add `dp`, `dpt` and `dpt=` to cover the rest of the plugin's functions?